asf_heat_pump_affordability/pipeline/preprocess_data.py

```diff
@@ -0,0 +1,36 @@
+def apply_exclusion_criteria(df, cost_year_min=None, cost_year_max=None):
+    """
+    Apply exclusion criteria to joined MCS-EPC dataframe to get analytical sample.
+
+    Exclusion criteria are rows where:
+        - Cost is NaN
+        - Tech type is NaN
+        - Tech type != Air Source Heat Pump
+        - No joined EPC record
+        - Insufficient property data to identify property archetype
+
+    Args
+        df (pd.DataFrame): joined MCS-EPC dataset
+        cost_year_min (int): min year of heat pump installation cost data to include
+        cost_year_max (int): max year of heat pump installation cost data to include
+
+    Returns
+        pd.DataFrame: Joined MCS and EPC dataset with analytical exclusion criteria applied.
+    """
+    if cost_year_min is not None:
```

Comment:
In both cases of this conditional, because `None` evaluates to `False` and numeric values other than 0 and strings other than "" (empty string) evaluate to `True`  you only need `if cost_year_min:`